tests/test_misc.py

```diff
@@ -315,6 +321,10 @@ def bitcoind_wait_new_block(bitcoind):
 @pytest.mark.skipif(
     not IS_NOT_BITCOIND_24, reason="Need 'generateblock' with 'submit=False'"
 )
+# TODO: Should this test work for other backends?
+@pytest.mark.skipif(
+    BITCOIN_BACKEND_TYPE is not BitcoinBackendType.Bitcoind, reason="FIXME"
+)
 def test_retry_on_workqueue_exceeded(lianad, bitcoind, executor):
```

Comment:
This test fails with electrs. I'm not sure if it's to do with how the fixtures are configured or something else. Am skipping it for now.

Reply:
i got it fail also on master on my machine
```
FAILED tests/test_misc.py::test_bitcoind_submit_block - AssertionError: b'\xfa\xbf\xb5\xdasendcmpct\x00\x00\x00\t\x00\x00\x00\xe9/^\xf8\x00\x02\x00\x00\x00\x00\x00\x00'
FAILED tests/test_misc.py::test_retry_on_workqueue_exceeded - AssertionError: b'\xfa\xbf\xb5\xdasendcmpct\x00\x00\x00\t\x00\x00\x00\xe9/^\xf8\x00\x02\x00\x00\x00\x00\x00\x00'

```

Reply:
Ah, so using bitcoind?

Reply:
yes not related to this PR

Reply:
This is a bitcoind-specific test.